environments/.env.template

```diff
@@ -21,13 +21,25 @@ KEYCLOAK_READER_PASSWORD="${KEYCLOAK_READER_PASSWORD}"
 KEYCLOAK_UPLOADER_PASSWORD="${KEYCLOAK_UPLOADER_PASSWORD}"
 
 # Variables only for deployment script
-BACKEND_DOCKERFILE="DockerfileBackend"
-FRONTEND_DOCKERFILE="DockerfileFrontend"
-KEYCLOAK_DOCKERFILE="DockerfileKeycloak"
+PROXY_IMAGE=ghcr.io/d-fine/dataland/dataland_inbound_proxy_production:${DATALAND_INBOUND_PROXY_PRODUCTION_VERSION}
```

Comment:
man könnte hier auch export vor allen Variablen schreiben dann könnte man sich das set allexport an anderer stelle sparen

Reply:
das allexport holt sich aber auch die VAriablen aus der GITHUB_ENV datei. 
Und in die kann man vermutlich nicht einfach export schreiben

Reply:
Wird im edc repo glaube ich aber so gemacht. Ich weiß aber nicht wie gut das funktioniert.

Reply:
Ja aber da müsste man ja nur das echo anpassen, siehe unten. Also mit anderen Worten überall wo set -o allexport benutzt wird werden danach Dateien gesourced die wir unter Kontrolle haben.

Reply:
Dann können wir es aber nicht mehr im  GITHUB_ENV benutzen - siehe
https://github.com/d-fine/Dataland/actions/runs/3428326830/jobs/5712452577